### PR TITLE
Update myFT URLs

### PR DESCRIPTION
**Pull request by keirog**

### CODE REVIEW

File: templates/overlay.html [hunk 1]
```diff
@@ -2,17 +2,17 @@
...
-			{{>n-myft-suggested-topics/main hLevel="3" text="Receive a daily digest email when stories on these topics are published.</br>You can change these any time at <a href=\"/myft/my-topics\" data-trackable=\"myft-topics\"><i>my</i>FT</a>"}}
+			{{>n-myft-suggested-topics/main hLevel="3" text="Receive a daily digest email when stories on these topics are published.</br>You can change these any time at <a href=\"/myft/explore\" data-trackable=\"myft-topics\"><i>my</i>FT</a>"}}
 			<button class="n-welcome__done" disabled data-trackable="done">Done</button>
 			<button class="n-welcome__later" data-trackable="remind-me-later">Remind me later</button>
 		</li>
 		<li class="n-welcome__slide" aria-hidden="true" data-slide="2">
 			<h2 class="n-welcome__header">Thanks!</h2>
 
-			<p class="n-welcome__text">Articles from topics you follow will appear in your myFT</br>
+			<p class="n-welcome__text">Articles from topics you follow will appear in your <i>my</i>FT</br>
```
Comment (matthew-andrews) [inline templates/overlay.html]: in your myFT?!  grammar police should be all over this one

Comment (laurieboyes) [inline templates/overlay.html]: will appear in your *my*FT
will appear in *my*FT
will appear in the *my*FT 
will appear in *your*FT

Hmm

**Comment by laurieboyes** [comment]
:+1:

File: templates/overlay.html (hunk 1 above)
Comment (matthew-andrews) [inline templates/overlay.html]: 我放弃说英语

Comment (keirog) [inline templates/overlay.html]: The grammar police’s tongue has been bitten off and its eyes blinded due to consistently high crime rates.

Comment (keirog) [inline templates/overlay.html]: (But thanks; this component needs so much love.)

**Pull request merged**